README.md

```diff
@@ -32,6 +32,15 @@ GF particularly addresses four aspects of grammars:
 
 ## Compilation and installation
 
+### Dependencies
+
+You need the C library `fcgi`[^1].  You also need to build the C
```

Comment:
Only if building the C version of the runtime, which is not the default

Reply:
What? Since when does the C run-time system depend on the fgci library?

Reply:
Just to clarify: I just jotted down what I had to do to make the project build (with `stack build`). I do not know the intricate details about this project. Just wanted to offer this up for your consideration.